### ISSUE TITLE
[solved] cocos.skeleton can traceback with AttributeError: 'TextureRegion' object has no attribute 'texture'

### ISSUE THREAD

**Issue by ccanepa** [los-cocos/cocos#323]
traceback while running test_skeleton_bitmap_skin.py

```
(py-cocos) E:\dev\cocos2019\test>python test_skeleton_bitmap_skin.py
Traceback (most recent call last):
  File "test_skeleton_bitmap_skin.py", line 43, in <module>
    main()
  File "test_skeleton_bitmap_skin.py", line 40, in main
    director.run(main_scene)
  File "..\cocos\director.py", line 406, in run
    event_loop.run()
  File "e:\dev\pyglet-2019\pyglet\app\base.py", line 175, in run
    self._run()
  File "e:\dev\pyglet-2019\pyglet\app\base.py", line 187, in _run
    timeout = self.idle()
  File "e:\dev\pyglet-2019\pyglet\app\base.py", line 314, in idle
    window.dispatch_event('on_draw')
  File "e:\dev\pyglet-2019\pyglet\window\__init__.py", line 1330, in dispatch_ev
ent
    if EventDispatcher.dispatch_event(self, *args) != False:
  File "e:\dev\pyglet-2019\pyglet\event.py", line 400, in dispatch_event
    if handler(*args):
  File "..\cocos\director.py", line 460, in on_draw
    self.scene.visit()
  File "..\cocos\cocosnode.py", line 741, in visit
    c.visit()
  File "..\cocos\cocosnode.py", line 741, in visit
    c.visit()
  File "..\cocos\cocosnode.py", line 734, in visit
    self.draw()
  File "..\cocos\skeleton.py", line 123, in draw
    self.blit_image(matrix, position, scale, image)
  File "..\cocos\skeleton.py", line 144, in blit_image
    a, b, _, c, d, _, e, f, _, g, h, _ = image.texture.tex_coords
AttributeError: 'TextureRegion' object has no attribute 'texture'
```

win7 64.bits, python 3.7

pyglet:
commit 58dfdb6314fc900cfbba6355b79bb09e718a41c9 master github
Author: Benjamin Moran
Date: Fri Aug 9 09:43:50 2019 +0900

cocos:
commit 26ef349 master
Author: Richard Jones
Date: Sun Aug 11 13:15:17 2019 +1000

**Comment by ccanepa** [comment]
No script in test\ show this traceback after the fix.

Keeping open while doing some other tests to keep in mind.

**Comment by ccanepa** [comment]
fixed in cocos 0.6.7
